add functions explorer panel in query builder (#888)
finos/legend-studio: .changeset/curly-swans-glow.md

---
'@finos/legend-graph': patch
---

> Added utility functions for function signature and call string generation.

Reviewer: ```suggestion
Added helper to generate function signature in text.
```
